_posts/2014-05-28-geomesa-quickstart.md

```diff
@@ -64,7 +65,7 @@ From within the root of the cloned tutorial, run:
 mvn clean install
 ```
 
-When this is complete, it should have built a JAR file that contains all of the code you need to run the tutorial.
+When this is complete, it should have built a jar file that contains all of the code you need to run the tutorial.
```

Comment:
Sorry to have been unclear on this point, but I would write ```a JAR file``` the way you had changed it previously, and ```geomesa-plugin-accumulo1.5-1.0.0-rc.4-SNAPSHOT-geoserver-plugin.jar``` as you have it now.  That is:  When used as an acronym, capitalize; when used as part of a file name, keep it lower-case (or however it needs to appear for the file system).  If that's confusing, please just holler.  Thanks!